### PR TITLE
feat(key list): add database keys listing, search and preview

### LINKED ISSUE #36: List database keys
It should be able to user list database keys when a database is selected in the sidebar. 

After connected, user can select a database from the sidebar and see all of its content. 

The database keys must be grouped for every `:` included in the key, like a graph. 

For example, the keys:
```
cache:provider:1
cache:provider:2
```
Will be converted in groups like:
```
cache > provider > keys 1 and 2
```
So if the user want to see the keys 1 and 2, he needs to click on cache group, then the provider and then the list of keys will be available.

It should also be able to user filter the keys by its name, a simple matching filter.

**Layout:** https://www.figma.com/file/YRor24p0TmTxcjl5L86jGb/Rocket-Redis?node-id=1%3A2

### PR DESCRIPTION
**Pull request by diego3g**
Add the key listing section when user selected a database. Also, the user can search through the
list. When the key get clicked, a preview is shown on the right sidebar, but not styled yet.

fix #36

### CODE REVIEW

File: src/screen/KeyList/styles.ts [hunk 1]
```diff
@@ -28,37 +33,68 @@ export const HeaderTitle = styled.p`
...
+interface KeyTextContainerProps {
+  selected: boolean
+}
+
+export const KeyTextContainer = styled.button<KeyTextContainerProps>`
   display: flex;
   flex-direction: row;
   align-items: center;
+  color: ${props =>
+    props.selected ? props.theme.colors.pink : props.theme.colors.white};
```
Comment (lenivene) [inline src/screen/KeyList/styles.ts:97]: Maybe it's good idea create method hover
Example:
```ts
&:hover{
color: props.theme.colors.pink;
opacity:.8;
}
```

File: src/screen/KeyList/index.tsx [hunk 2]
```diff
@@ -1,73 +1,152 @@
...
+    return keys.filter(key => key.includes(filter))
+  }, [filter, keys])
+
+  const rowVirtualizer = useVirtual({
+    size: filteredKeys.length,
+    parentRef,
+    estimateSize: useCallback(() => 33, [])
+  })
+
+  const handleSearchInputChange = useCallback(
```
Comment (lenivene) [inline src/screen/KeyList/index.tsx:72]: I think would be a good idea to be real-time search.

For example: I create a new key this moment, but if me search now (connected before creating), no have data.

File: src/screen/KeyList/index.tsx [hunk 3]
```diff
@@ -1,73 +1,152 @@
...
+    },
+    []
+  )
+
+  const handleSelectKey = useCallback((key: string) => {
+    setCurrentKey(key)
+  }, [])
+
+  useEffect(() => {
+    if (currentDatabase) {
```
Comment (YegorZaremba) [inline src/screen/KeyList/index.tsx:84]: I would like add 
```
setCurrentKey(undefined)
```
above to reset content value, its triggering me, but maybe its a feature

Comment (diego3g) [reply src/screen/KeyList/index.tsx:89]: Fixed.

File: src/screen/KeyList/styles.ts (hunk 1 above)
Comment (diego3g) [reply src/screen/KeyList/styles.ts]: Fixed.

File: src/screen/KeyList/index.tsx (hunk 2 above)
Comment (diego3g) [reply src/screen/KeyList/index.tsx:74]: Fow now, i'll leave it without realtime, i think it would be great to have a refresh button instead of realtime.

Comment (lenivene) [reply src/screen/KeyList/index.tsx:74]: @diego3g great idea

**Comment by diego3g** [comment]
> Maybe we should translate the toast messages as well.

Yeah, i'll leave that to another PR as this is not related to this PR at all.

Thanks <3

**Pull request merged**